lib/index.js

```diff
@@ -54,11 +54,6 @@ exports.register = (server, options, next) => {
       headers['X-Rate-Limit-Remaining'] = rate.remaining;
       headers['X-Rate-Limit-Limit'] = rate.limit;
       headers['X-Rate-Limit-Reset'] = rate.reset;
-
-      // legacy headers
-      headers['rate-limit-remaining'] = rate.remaining;
```

Comment:
We are going to add a plugin in API to add these legacy headers until we can deprecate them completely. That makes this plugin more appropriate for open source use.